.gitignore

```diff
@@ -183,10 +183,6 @@ tags
 
 ### VisualStudioCode ###
 .vscode/*
```

Comment:
I think need to remove `*` because it excludes folder but include specific subfolder
But you don't need any files in .vscode ever.
What do you think Sviatoslav? I'm right?

Reply:
It doesn't really matter. Git doesn't track folders at all. So if you ignore all files under a folder, the folder itself won't make it to Git.